pl/python_introduction/README.md

```diff
@@ -12,13 +12,18 @@ Aby rozpocząć zabawę z Pythonem, musisz otworzyć jego *wiersz poleceń* na s
 
 Gdy już będziesz gotowa, postępuj według instrukcji poniżej.
 
-Chcemy otworzyć konsolę Pythona. Wpisz `python`, jeśli pracujesz na Windowsie, lub `python3`, jeśli pracujesz na MacOS/Linuxie. Wciśnij `Enter`.
+Chcemy otworzyć konsolę Pythona. 
+
+Wpisz `python`, jeśli pracujesz na Windowsie, lub `python3`, jeśli pracujesz na MacOS/Linuxie. Wciśnij `Enter`.
 
     $ python3
     Python 3.4.3 (...)
```

Comment:
Hi @ArkadioG,

actually I was trying to figure out, how do translations work with Github and Crowdin and I believe we have two options here:
 * abandon Crowdin at all,
 * move all translation work to Crowdin and use pull requests only for changes in our source language (meaning English).

EDIT: I saw that you answered on slack, so I will move this conversation (about Github/Crowdin issue) there.